### PR TITLE
Upgrade Ruby from 2.5.1 to 2.5.3

### PR DESCRIPTION
**Pull request by monfresh**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ jobs:
       # The first image listed is the image that all steps run inside of.
       # This can be modified by steps, or can be built as a separate
       # customized container.
-      - image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/circleci:ruby-2.5.1
+      - image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/circleci:ruby-2.5.3-node-browsers
```
Comment (nemodjuric) [inline .circleci/config.yml]: Image URI should be 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/circleci:2.5.3-node-browsers

Comment (monfresh) [reply .circleci/config.yml]: Yeah. I didn't change the lint and js_tests parts yet. The rspec job is set to the correct image, but it's still failing. I will change the others as well to see if it makes a difference.

File: .circleci/config.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ jobs:
       # The first image listed is the image that all steps run inside of.
       # This can be modified by steps, or can be built as a separate
       # customized container.
-      - image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/circleci:ruby-2.5.1
+      - image: 008577686731.dkr.ecr.us-gov-west-1.amazonaws.com/circleci:2.5.3-node-browsers
```
Comment (enriquemanuel) [inline .circleci/config.yml]: my only concern with this is that now we are breaking naming convention (if we ever had one)

File: .circleci/config.yml [hunk 3]
```diff
@@ -150,7 +150,7 @@ jobs:
...
```
Comment (enriquemanuel) [inline .circleci/config.yml]: my only concern with this is that now we are breaking naming convention (if we ever had one)

File: .circleci/config.yml [hunk 4]
```diff
@@ -212,7 +212,7 @@ jobs:
...
```
Comment (enriquemanuel) [inline .circleci/config.yml]: my only concern with this is that now we are breaking naming convention (if we ever had one)

**Pull request merged**